dev/devicelab/bin/tasks/flavors_test_ios.dart

```diff
@@ -97,3 +100,54 @@ Future<TaskResult> _testInstallBogusFlavor() async {
 
   return TaskResult.success(null);
 }
+
+Future<TaskResult> _testFlavorWhenBuiltFromXcode(String projectDir) async {
+  final Device device = await devices.workingDevice;
+  await inDirectory(projectDir, () async {
+    await flutter('clean');
```

Comment:
Just curious, why was this needed?

If you know, please add a comment.

Reply:
It was to ensure the app bundle was left over from the previous parts of the test. I changed it to just delete the app bundle since that's faster the `flutter clean`